### PR TITLE
Checkout: Remove Payment Method a/b test

### PR DESCRIPTION
**Pull request by fditrapani**
#### Changes proposed in this Pull Request

This PR removes the payment method a/b test documented here: p8Eqe3-RL-P2. The test is over and due to the temporary solution for implementing it as well as the results being flat-ish, I have opted to keep the tabs as is.

#### Testing instructions
* Add a product to our cart and go to checkout. 
* You should only see the tabs and no radio buttons.

### CODE REVIEW

**Comment by sirbrillig** [comment]
Code looks good, and things work correctly. There's an odd styling issue that I'm not sure is correct, though:

tabs

The way that tab is highlighted seems wrong. It's different from the way the PayPal tab gets highlighted also.

Screen Shot 2019-11-08 at 6 28 41 PM

File: client/my-sites/checkout/checkout/style.scss [hunk 1]
```diff
@@ -1187,118 +1187,3 @@
...
-.payment-box__pm-wrapper {
-	margin: 0;
-	padding: 0;
-
-	.section-nav-tab {
-		list-style: none;
-		border: 0;
-		margin: 8px 0 0;
-		position: relative;
-		z-index: z-index( 'root', '.payment-box__pm-wrapper .section-nav-tab' );
```
Comment (Aurorum) [inline client/my-sites/checkout/checkout/style.scss]: Would it not also be worth removing the z-index registered here: https://github.com/Automattic/wp-calypso/blob/4db3529a9bb9c6869e6587dc91a3acbf1ab732eb/client/assets/stylesheets/shared/functions/_z-index.scss#L68

File: client/my-sites/checkout/checkout/style.scss [hunk 2]
```diff
@@ -1187,118 +1187,3 @@
...
-			position: absolute;
-			top: 50%;
-			transform: translateY( -50% );
-			left: 15px;
-			width: 21px;
-			height: 21px;
-			border: 1px solid var( --color-neutral-10 );
-			border-radius: 50%;
-			box-sizing: border-box;
-			z-index: z-index( 'root', '.payment-box__pm-wrapper .section-nav-tab::after' );
```
Comment (Aurorum) [inline client/my-sites/checkout/checkout/style.scss]: And here! https://github.com/Automattic/wp-calypso/blob/4db3529a9bb9c6869e6587dc91a3acbf1ab732eb/client/assets/stylesheets/shared/functions/_z-index.scss#L38

File: client/my-sites/checkout/checkout/style.scss (hunk 1 above)
Comment (fditrapani) [reply client/my-sites/checkout/checkout/style.scss]: Thanks for the catch. I'll take it out.

File: client/my-sites/checkout/checkout/style.scss (hunk 2 above)
Comment (fditrapani) [reply client/my-sites/checkout/checkout/style.scss]: Thanks for the catch. I'll take it out.

**Comment by fditrapani** [comment]
> There's an odd styling issue that I'm not sure is correct, though:

I noticed this too @sirbrillig. I couldn't figure it out in the couple minutes I had to review it so I left it out and thought I'd deal with it in another PR. I'll merge this one when the tests pass and deal with the style separately.

**Comment by fditrapani** [comment]
@sirbrillig here's a PR to fix that tab issue: https://github.com/Automattic/wp-calypso/pull/37525

**Pull request merged**